content-services/content-sources-frontend
# Fixes 3653: manual snapshot trigger disabled for specific repos

## Summary

- Disables the `Trigger snapshot` action if a repo does not have snapshotting enabled and adds a tooltip to alert the user

## Testing steps

- Create a repository without snapshotting enabled
- Upon clicking the kebab for that repository, you should see the `Trigger snapshot` action disabled and a tooltip on that action that says `Snapshots disabled for this repository.`
